### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DominikTher**
Series and Security

### CODE REVIEW

File: DT.Business.Entities/Series.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ public class Series
     {
         public int Id { get; set; }
         public double Weight { get; set; }
+        public int Repetitions { get; set; }
```
Comment (IriZnoj) [inline DT.Business.Entities/Series.cs:11]: will be integer enough? :D

File: DT.DataRepository/Repositories/AppUserRepository.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        private readonly Func<WorkoutContext> contextFactory;
+
+        public AppUserRepository(Func<WorkoutContext> contextFactory)
+            : base(contextFactory) => this.contextFactory = contextFactory;
+
+        public AppUser GetAppUser(string email, string password)
+        {
+            var dbContext = contextFactory.Invoke();
+
+            return dbContext.AppUsers.FirstOrDefault(appUser => appUser.Email == email && appUser.Password == password);
```
Comment (IriZnoj) [inline DT.DataRepository/Repositories/AppUserRepository.cs:21]: You should hash password!!! and not save it in plain text, oh maaan

File: DT.Client.Entities/AppUserAuth.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        public string Name { get; set; }
+        public string Token { get; set; }
+        public string Email { get; set; }
+        public bool IsAuthenticated { get; set; }
+
+        public AppUserAuth()
+        {
+            Name = "Not authorized";
+            Token = string.Empty;
```
Comment (IriZnoj) [inline DT.Client.Entities/AppUserAuth.cs:17]: You are not having Refresh Token? It is not secure to send login and password again and again. It should be done only one and than Refresh Token should be used

File: DT.Client.WebAPI/appsettings.json [hunk 4]
```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "SQLite": "Data Source=workout.db"
```
Comment (IriZnoj) [inline DT.Client.WebAPI/appsettings.json:11]: you are exposing db file or it is empty? ↓

File: DT.DataRepository/Repositories/AppUserRepository.cs (hunk 2 above)
Comment (DominikTher) [reply DT.DataRepository/Repositories/AppUserRepository.cs:21]: Will be implemented. Don't worry

File: DT.Business.Entities/Series.cs (hunk 1 above)
Comment (DominikTher) [reply DT.Business.Entities/Series.cs:11]: I am not sure too, just initial setup

File: DT.Client.WebAPI/appsettings.json (hunk 4 above)
Comment (DominikTher) [reply DT.Client.WebAPI/appsettings.json:11]: This is only connection string. DB file is right now in repo too but will be removed forever soon.

File: DT.Client.Entities/AppUserAuth.cs (hunk 3 above)
Comment (DominikTher) [reply DT.Client.Entities/AppUserAuth.cs:17]: Thank you, I will check it!

**Pull request merged**